Add severity allowlist to sentry integration installation (#8182)
PostHog/posthog.com: contents/docs/libraries/sentry.md

Make sure you're using both PostHog and Sentry as JS modules. You'll need to replace `'your organization'` and `project-id` with the organization and project-id from Sentry.

- `'your organization'`: will be in the URL when you go to your Sentry instance, like so: `https://sentry.io/organizations/your-organization/projects/`
- `project-id`: will be the last few digits in your Sentry DSN, such as `https://adf90sdc09asfd3@9ads0fue.ingest.sentry.io/project-id`
- `prefix`: Optional: Url of a self-hosted sentry instance (default: https://sentry.io/organizations/)

> - `severityAllowList`: Optional: by default this is ['error'], you can provide more Sentry severity levels (e.g. ['error', 'info']) or '*' to capture all messages. Only availbl from posthog-js version 1.118.0 forward

Author: ```suggestion
- `severityAllowList`: Optional: by default this is ['error'], you can provide more Sentry severity levels (e.g. ['error', 'info']) or '*' to capture all messages. Only available from posthog-js version 1.118.0 forward
```

> - `severityAllowList`: Optional: by default this is ['error'], you can provide more Sentry severity levels (e.g. ['error', 'info']) or '*' to capture all messages. Only available from posthog-js version 1.118.0 forward

Author: ```suggestion
- `severityAllowList`: Optional: by default this is ['error'], you can provide more Sentry severity levels (e.g. ['error', 'info']) or '*' to capture any severity. Only available from posthog-js version 1.118.0 forward
```

Sentry.init({
    dsn: '<your Sentry DSN>',
    integrations: [new posthog.SentryIntegration(
        posthog,
        'your organization',
        project-id,
        undefined, // optional: but necessary if you want to set a severity allowlist

>         ['error', 'info'] // optional: will react to captureMessage (info) and captureException (error)

Author: ```suggestion
        ['error', 'info'] // optional: here is set to handle captureMessage (info) and captureException (error)
```
